genai-perf/README.md

```diff
@@ -128,7 +128,7 @@ the GPT-2 model running on Triton Inference Server with a TensorRT-LLM engine.
 ### Serve GPT-2 TensorRT-LLM model using Triton CLI
 
 You can follow the [quickstart guide](https://github.com/triton-inference-server/triton_cli?tab=readme-ov-file#serving-a-trt-llm-model)
-on Triton CLI github repo to run GPT-2 model locally.
+in Triton CLI github repository to serve GPT-2 model on the Triton server with TensorRT-LLM backend.
```

Comment:
```suggestion
in the Triton CLI GitHub repository to serve GPT-2 on the Triton server with the TensorRT-LLM backend.
```

Could you update the capitalization for GitHub everywhere this phrase is used? And add the articles (e.g. "the")?

Reply:
sure, done!